db/schema.rb

```diff
@@ -90,24 +90,24 @@
   end
 
   create_table "file_attachment_assets", force: :cascade do |t|
-    t.bigint "file_revision_id", null: false
+    t.bigint "blob_revision_id", null: false
     t.string "variant", default: "file", null: false
     t.string "file_url"
     t.string "state", default: "absent", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["file_revision_id", "variant"], name: "index_file_attachment_assets_on_file_revision_id_and_variant", unique: true
-    t.index ["file_revision_id"], name: "index_file_attachment_assets_on_file_revision_id"
+    t.index ["blob_revision_id", "variant"], name: "index_file_attachment_assets_on_blob_revision_id_and_variant", unique: true
+    t.index ["blob_revision_id"], name: "index_file_attachment_assets_on_blob_revision_id"
     t.index ["file_url"], name: "index_file_attachment_assets_on_file_url", unique: true
   end
 
-  create_table "file_attachment_file_revisions", force: :cascade do |t|
+  create_table "file_attachment_blob_revisions", force: :cascade do |t|
     t.bigint "blob_id", null: false
     t.bigint "created_by_id"
     t.string "filename", null: false
     t.datetime "created_at", null: false
     t.bigint "size", null: false
-    t.index ["blob_id"], name: "index_file_attachment_file_revisions_on_blob_id"
+    t.index ["blob_id"], name: "index_file_attachment_blob_revisions_on_blob_id"
```

Comment:
it doesn't look like there are migrations that rename this, and any other, indexes?

Reply:
I doubted myself there for a moment, but thankfully this is done as part of renaming the column.

```
psql content_publisher_development -c '\di' | grep file_revision
```

Reply:
Oh TIL didn't realise the rename commands had extra side effects